### ISSUE TITLE
No name is defined for "How can I write unit tests in visual studio code: A11y_VS Code extension for API Center_Copilot Chat_Name Role Value.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-azureapicenter#106]
### GitHub Tags
#A11yTCS; #A11ySev2; #Mac; #WCAG4.1.2; #GH_VSCodeextensionforAPICenter_Mac_Apr2024; #DesktopApp; #VS Code Extension for API Center; #Name Role Value; #AILimited; #A11yMAS; #device;#FirstParty:Bug212321;#External;#External:VSCode;#BM-VisualStudioCodeClient-Win32-Jan2024;

### Environment Details:
Application Name: VS Code Extension for API Center
macos sonoma version:14.4.1

### Repro Steps:

1. Create a API center in azure portal.
2. Open Visual studio Code.
3. Tab till Chat option in the left side panel and press enter.
4. Type @workspace /tests in the copilot chat box.
5. Verify whether the name is defined for "No name is defined for "How can I write unit tests in visual studio code" button or not.

### Actual Result:
No name is defined for "How can I write unit tests in visual studio code.

### Expected Result:
Name should be defined for "How can I write unit tests in visual studio code." button.

### User Impact:
Screen reader users will not get the information about the functionality of the button if name is not defined.

### Attachment:
<img width="1032" alt="Bug_106" src="https://github.com/microsoft/vscode-azureapicenter/assets/124244478/fe9f7dcb-cae2-4ba0-85b0-ebfe3af89efc">

**Comment by formulahendry** [comment]
External issue. This UI control is from GitHub Copilot Chat: https://github.com/microsoft/vscode-copilot-release

**Comment by Yash14j** [comment]
@formulahendry As per the C+AI accessibility testing external bug process primary team need to file the bug to VSCode team & share us the external bug ID.

**Comment by formulahendry** [comment]
This issue is caused by UI of GitHub Copilot Chat extension, not related to UI of API Center extension.

**Comment by formulahendry** [comment]
Opened bug in vs code repo: https://github.com/microsoft/vscode/issues/212321
